src/main/java/com/develhope/spring/order/dto/OrderMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.develhope.spring.order.dto;
+
+import com.develhope.spring.order.entity.Order;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OrderMapper {
+    public static OrderDTO toDTO(Order order) {
+        OrderDTO dto = new OrderDTO();
+        dto.setOrderStatus(order.getOrderStatus());
+        dto.setVehicleId(order.getUser().getId());
```

Comment:
Controllerei se Vehicle è presente e in caso contrario lanciamo un eccezione

Reply:
ma nel dto?